src/components/FlawForm.vue

```diff
@@ -87,7 +87,8 @@ const errors = {
   cwe_id: null,
   major_incident_state: null,
   reported_dt: null,
-  unembargo_dt: null,
+  embargo_old_public_dt: 'Set the current or a future date for an embargoed flaw.',
+  unembargo_future_public_dt: 'Set the current or an previous date for unembargoed flaw.',
```

Comment:
I think it would be simpler to return the strings from the validation method.

Reply:
I just defined them in the _const errors_ variable because I saw it was already defined.